<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="btnAction">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>50</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCfg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>姿态配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInfo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBig">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSmall">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>731</width>
     <height>561</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_config">
    <attribute name="title">
     <string>姿态配置</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_game">
    <attribute name="title">
     <string>游戏</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_bang">
    <attribute name="title">
     <string>排行榜</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QFrame" name="frameDet">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>450</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelGame">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPose">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>220</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>289</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>摄像头配置</string>
   </property>
   <widget class="QComboBox" name="comboCamera">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioFlip">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>57</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>翻转</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCamera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFlip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>画面翻转</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
